packages/integrations/sitemap/README.md

```diff
@@ -129,66 +129,69 @@ Sitemap: https://<YOUR SITE>/sitemap-index.xml
 
 To configure this integration, pass an object to the `sitemap()` function call in `astro.config.mjs`.
 
-__`astro.config.mjs`__
+**`astro.config.mjs`**
 
 ```js
 import { defineConfig } from 'astro/config';
 import sitemap from '@astrojs/sitemap';
 
 export default defineConfig({
-  integrations: [sitemap({
-    filter: ...
-  })]
+  integrations: [
+    sitemap({
+      filter, // ...
+    }),
+  ],
```

Comment:
Need to use a slightly different syntax so it's valid JS. Was also thinking to remove `filter,` entirely and keep `// ...` only

Reply:
That would work — or something like `// configuration options` to be clear what it’s a placeholder for?